display/projectiles.py

```diff
@@ -29,10 +35,13 @@ def __init__(self, icon, velocity, oscillate_height = 0, oscillate_rate = 1):
 
     def draw_image(self):
         ifile = pygame.image.load(os.path.join("assets", "items", self.icon + ".png"))
-        self.image = pygame.transform.scale(ifile, (150, 150))
+        self.image = pygame.transform.scale(ifile, (15*Projectile.pixel_scale, 15*Projectile.pixel_scale))
 
     def update(self):
         self.rect.x += self.velocity
 
         if self.oscillate_height != 0:
             self.rect.y = self.default_y + math.floor(math.sin((self.rect.x * self.oscillate_rate / screen_width) * 2 * math.pi) * self.oscillate_height)
+
+        if self.rect.x < -500 or self.rect.x > screen_width+500:
```

Comment:
Ew, lack of spaces :nauseated_face:

Reply:
here:                    have some more.